lib/elixir_bench/benchmarks/job.ex

```diff
@@ -46,6 +46,7 @@ defmodule ElixirBench.Benchmarks.Job do
   @create_fields [
     :branch_name,
     :commit_sha,
+    :repo_id,
```

Comment:
it seems weird that we'd need to add this here the application was working before/is working - why would we need to add it here? :thinking: